packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.tsx

```diff
@@ -51,6 +52,16 @@ export function ComboboxContentList(props: ComboboxListProps): JSX.Element {
               ))}
             </>
           )}
+          {props.onLoadMore && (
+            <ComboboxLoadMore
+              onLoadMore={props.onLoadMore}
+              loading={props.loading}
+            >
+              <div className={styles.loadingContainer}>
+                <Spinner size="small" />
+              </div>
+            </ComboboxLoadMore>
+          )}
```

Comment:
So this implementation couples the loading state to the `onLoadMore`. I get it tho, it does feel like there's 3 states for the loading state.

But there's actually only 2. 
- If it's loading and there are no options (Glimmer)<br/>![image](https://github.com/GetJobber/atlantis/assets/15986172/94edbf27-9c95-4745-a173-e4d796ef80d2)
- If it's loading and there are options (Spinner stuck to the bottom and always visible)<br/>![image](https://github.com/GetJobber/atlantis/assets/15986172/e2f88fef-8652-4fc3-9863-29f46d3589c2)

Both scenarios don't need to know if you have an `onLoadMore` (or `onSearch` if that doesn't work as expected (it probably isn't looking at where it is in the code 😆))

Reply:
Perfect. Love it.